test/extended/builds/multistage.go

```diff
@@ -23,10 +23,10 @@ var _ = g.Describe("[sig-builds][Feature:Builds] Multi-stage image builds", func
 		testDockerfile = `
 FROM scratch as test
 USER 1001
-FROM registry.redhat.io/rhel7
+FROM image-registry.openshift-image-registry.svc:5000/openshift/cli:latest as other
```

Comment:
Not sure if this matters, but the above line that was replaced did not have the `as other`

Reply:
Yeah that was intentional @coreydaley ... in the 4.7 changes the `as other` was added by Clatyon

Reply:
`as other` is an alias in a Dockerfile. ok to keep for a multi-stage build test.

Reply:
got it, thanks for the clarification.

Reply:
For reference the 4.7 flavor:  https://github.com/openshift/origin/blob/4cbb844d1d5f3a3ac6dd08325db17b807e7129f2/test/extended/builds/multistage.go#L27